*** cuml/python/setup.py-dist	2021-08-04 23:26:19.434606791 +0900
--- cuml/python/setup.py	2021-08-04 23:28:48.687328829 +0900
***************
*** 151,157 ****
          # object has all the args used by the user, we can check that.
          self.singlegpu = '--singlegpu' in self.distribution.script_args
  
!         libs = ['cuml++', 'cudart', 'cusparse', 'cusolver']
  
          include_dirs = [
              '../cpp/src',
--- 151,161 ----
          # object has all the args used by the user, we can check that.
          self.singlegpu = '--singlegpu' in self.distribution.script_args
  
!         libs = ['cuml++', 'cuda', 'cudart',
!                 'cublas', 'cublasLt', 'cufft', 'cufftw', 'cuinj64',
!                  'curand', 'cusolver', 'cusolverMg', 'cusparse', 'faiss_avx2',
!                  'treelite', 'fmt']
! 
  
          include_dirs = [
              '../cpp/src',
*** cuml/python/cuml/manifold/umap.pyx-dist	2022-01-25 22:55:33.931640327 +0900
--- cuml/python/cuml/manifold/umap.pyx	2022-01-25 23:14:48.295407566 +0900
***************
*** 373,383 ****
          else:
              raise Exception("Initialization strategy not supported: %d" % init)
  
!         if a is None or b is None:
!             a, b = self.find_ab_params(spread, min_dist)
! 
!         self.a = a
!         self.b = b
  
          self.learning_rate = learning_rate
          self.min_dist = min_dist
--- 373,385 ----
          else:
              raise Exception("Initialization strategy not supported: %d" % init)
  
!         if a is None:
!             self.a, self.b = self.find_ab_params(spread, min_dist)
!         elif b is None:
!             self.a, self.b = self.find_ab_params(spread, min_dist)
!         else:
!             self.a = a
!             self.b = b
  
          self.learning_rate = learning_rate
          self.min_dist = min_dist
***************
*** 470,476 ****
          free(umap_params)
  
      @staticmethod
!     def find_ab_params(spread, min_dist):
          """ Function taken from UMAP-learn : https://github.com/lmcinnes/umap
          Fit a, b params for the differentiable curve used in lower
          dimensional fuzzy simplicial complex construction. We want the
--- 472,478 ----
          free(umap_params)
  
      @staticmethod
!     def find_ab_params(self, spread, min_dist):
          """ Function taken from UMAP-learn : https://github.com/lmcinnes/umap
          Fit a, b params for the differentiable curve used in lower
          dimensional fuzzy simplicial complex construction. We want the
